treeherder/seta/management/commands/reset_seta.py

```diff
@@ -1,13 +1,18 @@
 from django.core.management.base import BaseCommand
```

Comment:
I'm not familiar enough with SETA to know if this is 100% correct, but my understanding is that we're trying to solve this problem:
* There are two tasks that get run, analyze_failures and load_preseed
* analyze_failures does different things depending on whether some table is empty or not
* analyze_failures needs to run at least once where it sets things as low priority (which it only does if the table is empty)
* load_preseed adds things to this table, making it non-empty
* therefore if load_preseed runs before analyze_failures does for the first time, then analyze_failures never does its "first run" type behaviour

It seems that there are 4 ways to fix this:
1) make load_preseed check to see if analyze_failures has ever been run, otherwise it just does nothing (thereby it can be safely be run at any point, and will bail with a warning if too soon)
2) make analyze_failures use a better way of determining whether to do the "first run" behaviour than checking if some table is empty
3) force analyze_failures to always run before load_preseed by always doing both at the same time
4) make analyze_failures delete everything that table before running itself and then load_preseed

Option 3 is what added minutes to the Vagrant provision and deploy times (bug 1329685).
Option 4 is the current PR's approach.

...which leaves option 1/2, or any others I've missed?

Perhaps now that @armenzg is back, he might know what's best to do?